openedx/adg/lms/applications/views.py

```diff
@@ -262,4 +264,29 @@ def get_context_data(self, request):
                     'birth_month': extended_profile.birth_date.month,
                     'birth_year': extended_profile.birth_date.year,
                 })
+
```

Comment:
@zeshanarifios `return context` of ContactInformationView->get_context_data has been removed.

Reply:
Nice catch!